app/display.c

```diff
@@ -83,39 +83,33 @@ void display_init(void)
     set_foreground_colour(BLACK);
     set_background_colour(WHITE);
     clear_screen_region(0, 0, 0, 27);
-    prints(0, 0, "     Memtest86+ v6.00b1");
+    prints(0, 5, "Memtest86+ v6.00b1");
     set_foreground_colour(RED);
     printc(0, 14, '+');
     set_foreground_colour(WHITE);
     set_background_colour(BLUE);
-    prints(0,28,                             "| ");
-    prints(1, 0, "CLK/Temp:   N/A             | Pass   % ");
-    prints(2, 0, "L1 Cache:   N/A             | Test   % ");
-    prints(3, 0, "L2 Cache:   N/A             | Test #   ");
-    prints(4, 0, "L3 Cache:   N/A             | Testing: ");
-    prints(5, 0, "Memory  :   N/A             | Pattern: ");
-    prints(6, 0, "--------------------------------------------------------------------------------");
```

Comment:
Comment the line rather than removing it, again because it documents the screen layout.

Reply:
Done.